bin/otomi

```diff
@@ -24,7 +24,7 @@ ENV_DIR=${ENV_DIR:-$PWD}
 
 . bin/common.sh
 
-[ $has_docker = 'false' ] && [[ ! "$CI" ]] && err "Otomi CLI expects a running docker daemon!" && exit 1
+[ $has_docker = 'false' ] && [ -n "$CI" ] && err "Otomi CLI expects a running docker daemon!" && exit 1
```

Comment:
now that I think of this, I think this second check should test for IN_DOCKER to be valuable, because even in CI we need docker to run

Reply:
Sorry, I don't fully understand what you mean by that...